testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -124,7 +126,7 @@ def test_strings(self):
         assert_equal(util.iterable(u"unicode string"), False)
 
 
-class TestFilename(object):
+class TestFilename(TestCase):
```

Comment:
You are using `TestCase` here to make the test classes discoverable to pytest. This means this class isn't normally picked up by pytest. Please make sure the class has no init method. You can also run pytest with `--collect-only` to make sure all tests are picked up quickly.